tests/test_solver_kamino.py

```diff
@@ -0,0 +1,142 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 The Newton Developers
+# SPDX-License-Identifier: Apache-2.0
+
+import unittest
+
+from pxr import Plug, Usd, UsdPhysics
+
+import newton_usd_schemas  # noqa: F401
+
+
+class TestNewtonKaminoSceneAPI(unittest.TestCase):
+    def setUp(self):
+        self.stage: Usd.Stage = Usd.Stage.CreateInMemory()
+        self.scene: Usd.Prim = UsdPhysics.Scene.Define(self.stage, "/Scene").GetPrim()
+
+    def test_api_registered(self):
+        plug_type = Plug.Registry().FindTypeByName("NewtonPhysicsKaminoSceneAPI")
+        self.assertEqual(plug_type.typeName, "NewtonPhysicsKaminoSceneAPI")
+        schema_type = Usd.SchemaRegistry().GetSchemaTypeName("NewtonPhysicsKaminoSceneAPI")
+        self.assertEqual(schema_type, "NewtonKaminoSceneAPI")
+
+    def test_api_application(self):
+        self.scene.ApplyAPI("NewtonKaminoSceneAPI")
+        self.assertTrue(self.scene.HasAPI("NewtonKaminoSceneAPI"))
```

Comment:
As in #18 lets add the extra check


```suggestion
        self.assertTrue(self.scene.HasAPI("NewtonSceneAPI"))
        self.assertTrue(self.scene.HasAPI("NewtonKaminoSceneAPI"))
```